packages/ui/src/assets/css/vars.css

```diff
@@ -1,19 +1,22 @@
 :root {
-  --border-color: #b5b5b5;
-  --color: rgb(84,84,84);
-  --dz-alternate-color: #0284c7;
-  --dz-danger-color: #dc3545;
+  --color: rgb(84, 84, 84);
   --dz-primary-color: #007aff;
   --dz-secondary-color: #475569;
-  --dz-success-color: #22C55E;
-  --dz-warning-color: #EA580C;
+  --dz-alternate-color: #0284c7;
+  --dz-danger-color: #dc3545;
+  --dz-success-color: #22c55e;
+  --dz-warning-color: #ea580c;
+  --field-autofill-color: #ffffff;
+  --field-background-color: var(--dz-primary-color);
+  --field-border-color: #ccc;
+  --field-border-radius: 0.5rem;
+  --field-invalid-border-color: var(--dz-danger-color);
+  --field-outline-color: var(--dz-primary-color);
+  --field-padding-height: 0.5em;
+  --field-valid-border-color: var(--dz-success-color);
+  --border-color: #b5b5b5;
   --error-message-color: var(--dz-danger-color);
-  --input-autofill-color: #ffffff;
-  --input-background-color: var(--dz-primary-color);
-  --input-invalid-border-color: #dc3545;
-  --input-outline-color: var(--dz-primary-color);
-  --input-valid-border-color: #198754;
```

Comment:
We need to make sure that these are not in use in any components. As far as I remember, we were still using it in some of the input components. Please check.